ts-serverless-quickstart/src/pets/cats.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @klotho::execution_unit {
+ *   id = "cats"
+ * }
+ */
+
+
+ export const router = require('express').Router();
+
+ function getCats(event, res) {
+     res.send("[Maine Coon, Ragdoll, Shorthair]");
+ }
+ 
+ router.get('/v1/cats', getCats);
+ 
```

Comment:
```suggestion

```

(super minor nit :) — I only noticed it because gh had the warning that there's no trailing newline)